### PR TITLE
Feature/event location

### PR DESCRIPTION
**Pull request by KCWill**
### Type of change made:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling (Styling to new or existing files)

### Detailed Description
Added create event functionality with POST. Still waiting on location POST functionality. Added live search bar functionality. Added tag filtering functionality.

### Why is this change required? What problem does it solve?
Allows user to create and post a new event. Allows user to filter and search event list.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:4000
```
Comment (collinkallery) [inline .env:1]: ENV variable - there are a few fetches currently that are NOT using this and we will need to change that, but the main fetch is working with this variable.

File: src/Actions/index.js [hunk 2]
```diff
@@ -55,4 +68,93 @@ export function removeFavorite(eventID) {
...
+      headers: myHeaders,
+      body: raw,
+      redirect: 'follow'
+    };
+    return fetch("http://localhost:4000/api/events", requestOptions)
+      .then(response => response.json())
+      .then(result => console.log('result', result))
+      .catch(error => console.log('error', error));
+  }
+}
```
Comment (collinkallery) [inline src/Actions/index.js:89]: Will need to add our type and payload to this?

File: src/Reducers/index.js [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
 
-const savedEvents = (state = initialState, action) => {
+const displayEventInitialState = {
+  eventList: [],
+  selectedEvent: {
+    location: {},
+    artists: [],
+    labels: []
+  }
+}
```
Comment (collinkallery) [inline src/Reducers/index.js:18]: ~ Y ~ A ~ S ~

File: src/Actions/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 export function getEvents() {
   return dispatch => {
-    return fetch('http://localhost:4000/api/events')
+    return fetch(process.env.REACT_APP_API_URL+'/api/events')
```
Comment (danielbldr) [inline src/Actions/index.js:3]: Yay glad we got this env variable to work

File: src/Actions/index.js [hunk 5]
```diff
@@ -55,4 +68,93 @@ export function removeFavorite(eventID) {
...
+      .then(response => response.json())
+      .then(result => console.log('result', result))
+      .catch(error => console.log('error', error));
+  }
+}
+
+
+
+  //
+  // return fetch(process.env.REACT_APP_API_URL+"/api/events", {
```
Comment (danielbldr) [inline src/Actions/index.js:112]: Was this a way for you to test this function works?

**Pull request merged**